Assets/UI/Popups/greatpeoplepopup.lua

```diff
@@ -146,302 +172,306 @@ function GetBiographyTextTable( individualID:number )
     return kBiography;
 end
 
-
 -- ===========================================================================
---  View the great people currently available (to be purchased)
+-- Add an instance of a Great Person "recruit" to the Great People Panel
 -- ===========================================================================
-function ViewCurrent( data:table )
-    if (data == nil) then
-        UI.DataError("GreatPeople attempting to view current timeline data but received NIL instead.");
-        return;
+function AddRecruit( kData:table, kPerson:table )
```

Comment:
The code diff will not do a great job here, but this is the chunk of code that previously existed in the _for loop_ that is in the ViewCurrent function.  The code was moved to this function to match the file produced by Firaxis, the code was otherwise not changed.

Reply:
seriously this diff is brutal...
**no code is changed from this point down to the next point I mark below, just moved.  scroll until my next comment on the subject.**